cisco/xrv/docker/launch.py

```diff
@@ -135,35 +135,12 @@ def bootstrap_config(self):
         self.wait_write("terminal length 0")
 
         self.wait_write("crypto key generate rsa")
-        # check if we are prompted to overwrite current keys
-        (ridx, match, res) = self.tn.expect(
-            [
-                b"How many bits in the modulus",
-                b"Do you really want to replace them",
-                b"^[^ ]+#",
-            ],
-            10,
-        )
-        if match:  # got a match!
-            if ridx == 0:
-                self.wait_write("2048", None)
-            elif ridx == 1:  # press return to get started, so we press return!
-                self.wait_write("no", None)
```

Comment:
Thanks, could you explain a little further, because I don't think I can see this behaviour?

Thanks for your review, appreciate it a lot.

I'll have to go over each platform and migrate them all to their Scrapli (and Scrapli Community) drivers. If it's ok I'll request another review when that's all good to go :+1: